src/otx/core/exporter/base.py

```diff
@@ -57,6 +57,7 @@ def __init__(
         pad_value: int = 0,
         swap_rgb: bool = False,
         output_names: list[str] | None = None,
```

Comment:
Could you update docstring for this?

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/4031/commits/00200f4d4811d82e86e371cb5dabd2541098e83c